google/cloud/storage/internal/rest_client.cc

```diff
@@ -59,16 +59,26 @@ std::string UrlEscapeString(std::string const& value) {
   return std::string(handle.MakeEscapedString(value).get());
 }
 
+bool IsMinNotSuccess(rest::HttpStatusCode code) {
+  return code >= rest::kMinNotSuccess;
+}
```

Comment:
Nit: It seems strange for something called "**Is**MinNotSuccess" to be implemented as "**>**= kMinNotSuccess".